docker-multinode/master.sh

```diff
@@ -30,13 +30,13 @@ kube::multinode::detect_lsb
 
 kube::multinode::turndown
 
-kube::multinode::bootstrap_daemon
+kube::multinode::restart_docker
```

Comment:
Kublet is started after reboot while etcd and flannel is not. I guess kublet exists with failure, becuase `  RESTART_POLICY=${RESTART_POLICY:-"on-failure"}`